### ISSUE TITLE
Did the 9.00 Exploit without the USB Stick

### ISSUE THREAD

**Issue by VirusBurst** [KameleonReloaded/900V4B#6]
TL;DR because I am shocked

- Started PS4 from coldboot
- Cried about failed fPKG updates in queue 
- Launched browser, your page was open and initiated GoldHENv2.0B
- Asked for the USB Stick, plugged it in, Notification comes and goes, presses OK and suddenly reboot
- Removed the USB before PS4 rebootet, shown a message with error code, suggest to "repair" the system via update
- yada yada, open browser and see the counter for success increased... sus...
- checked setting for GoldHEN or Debug but nothing there so... was a fail
- tried again to jailbreak, failed two time due "not enough memory" and the third time... IMMEDIATELY GoldHEN loaded...
- /me still holding the USB Stick on hand and be like... "WTH happened?"

So... did I broke the jailbreak or everything fine?

**Comment by KameleonReloaded** [comment]
You did a full reboot cycle?

**Comment by VirusBurst** [comment]
No I didn't, it just "crashed" or whatever, showed the PS-Logo all the sudden, followed by the error message (not talking about the message I regular see about reporting something to Sony!)
